src/UserStory/SingleStory.js

```diff
@@ -0,0 +1,50 @@
+import React, { Component } from 'react'
+import { connect } from 'react-redux'
+import { deletestory } from './Actions/deleteStory'
+
+class story extends Component {
+
+    handleClick = () => {
+        this.props.deletestory(this.props.story.id)
+        this.props.history.push('/');
+    }
+
+    render(){
+        console.log(this.props)
+        const story = this.props.story ? (
```

Comment:
should destructure properties of `props` so that it is shorter and more readable. Ex: `const { story } = this.props`. The best practice here is to keep your extractor as short as possible (only one dot `.`)

Reply:
destructed the prop